pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -204,8 +204,8 @@ func (m *kubeGenericRuntimeManager) startContainer(podSandboxID string, podSandb
 	// to create it in the first place. it happens when journald logging driver is used with docker.
 	if _, err := m.osInterface.Stat(containerLog); !os.IsNotExist(err) {
 		if err := m.osInterface.Symlink(containerLog, legacySymlink); err != nil {
-			klog.Errorf("Failed to create legacy symbolic link %q to container %q log %q: %v",
-				legacySymlink, containerID, containerLog, err)
+			klog.ErrorS(err, "Failed to create legacy symbolic link", "legacySymlink", legacySymlink,
```

Comment:
what value is stored under `legacySymlink` variable? I think that key name should represent meaning behind a value. In this case I would expect that this variable stores "path to file where symlink exist", if that is the case I think better key would be `path`.

If this `legacySymlink` is not represented by path, but some other way to identify symlink then I would change it key from `legacySymlink` to `symlink` so that all logs referring to symlink use same key, no matter if in some context is legacy.

Reply:
Sorry for reply too later, Recently we were in the Chinese New Year holiday. I have changed the code.